public/index.html

```diff
@@ -1,34 +1,40 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <!-- <link rel="stylesheet" href="//node_modules/react-datepicker/dist/react-datepicker-cssmodules.css"/> -->
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <title>Page Title</title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <script src="https://kit.fontawesome.com/2475edd293.js"></script>
+    <script src="//baltimorecountymd.gov/sebin/z/l/dotgov-site.min.js"></script>
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://beta.baltimorecountymd.gov/sebin/j/e/dotgov.min.css"
+    />
     <style>
-      .plus-text-icon,
-      .minus-text-icon {
+         .dg_date-container1  {
         position: relative;
-      }
-      .plus-text-icon:before,
-      .minus-text-icon:before {
-        display: inline-block;
-        margin-right: 5px;
-      }
+        min-width: 40%;
+        margin-right:20px;
+         }
 
-      .plus-text-icon:before {
-        content: "+";
-      }
-
-      .minus-text-icon:before {
-        content: "-";
-      }
+          .dg_date-btn1 {
+            display: block;
+            position: relative;
+            right: -230px;
+            top: -35px;
+            cursor: pointer;
+          }
     </style>
```

Comment:
The styles from master need to come back. Those are used internal to this project. 

 <style>	    <style>
      .plus-text-icon,	         .dg_date-container1  {
      .minus-text-icon {	
        position: relative;	        position: relative;
      }	        min-width: 40%;
      .plus-text-icon:before,	        margin-right:20px;
      .minus-text-icon:before {	         }
        display: inline-block;	
        margin-right: 5px;	
      }	


      .plus-text-icon:before {	          .dg_date-btn1 {
        content: "+";	            display: block;
      }	            position: relative;

            right: -230px;
      .minus-text-icon:before {	            top: -35px;
        content: "-";	            cursor: pointer;
      }	          }
    </style>

Just cope that back over the style tag in your branch